core/src/services/gcs/backend.rs

```diff
@@ -66,6 +66,14 @@ pub struct GcsConfig {
     pub predefined_acl: Option<String>,
     /// The default storage class used by gcs.
     pub default_storage_class: Option<String>,
+    /// Explicitly allow anonymous access, such as for publicly available buckets
```

Comment:
Hi, the `allow` in `allow_anonymous` occurs after attempting to load the credentials.